Promises.podspec

```diff
@@ -1,6 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = "Promises"
   s.version          = "2.0.2"
+  s.swift_version    = "5.0"
```

Comment:
@epologee Can you tell me a little bit about why you added this? We can keep it in, but it will mean that nobody using a version of cocoapods before 1.7 can use the library.

Reply:
🤔 I was having the problem from the other direction, which was the trigger to create this PR. When running `pod install` on the latest CocoaPods wihout the `s.swift_version` in the Podfile, you get this (fatal) error:

```
[!] Unable to determine Swift version for the following pods:
 
- `Promises` does not specify a Swift version and none of the targets (`~~~`) 
  integrating it have the `SWIFT_VERSION` attribute set. Please contact the 
  author or set the `SWIFT_VERSION` attribute in at least one of the targets 
  that integrate this pod.
```

I'm not sure what the solution needs to be to support both the old and the new CocoaPods 😞...

Reply:
Is upgrading your cocoapods version an option?